ddprof-lib/src/main/cpp/j9Ext.cpp

```diff
@@ -54,27 +54,6 @@ bool J9Ext::initialize(jvmtiEnv* jvmti, const void* j9thread_self) {
        jvmti->Deallocate((unsigned char*)ext_functions);
     }
 
-    jvmtiExtensionEventInfo* ext_events;
-    if (jvmti->GetExtensionEvents(&ext_count, &ext_events) == 0) {
-        for (int i = 0; i < ext_count; i++) {
-            if (strcmp(ext_events[i].id, "com.ibm.InstrumentableObjectAlloc") == 0) {
-                InstrumentableObjectAlloc_id = ext_events[i].extension_event_index;
-                // If we don't set a callback now, we won't be able to enable it later in runtime
-                jvmti->SetExtensionEventCallback(InstrumentableObjectAlloc_id, (jvmtiExtensionEvent)J9ObjectSampler::JavaObjectAlloc);
-                jvmti->SetExtensionEventCallback(InstrumentableObjectAlloc_id, NULL);
-                break;
-            }
-        }
-       jvmti->Deallocate((unsigned char*)ext_events);
-    }
-
     return _GetOSThreadID != NULL && _GetStackTraceExtended != NULL && _GetAllStackTracesExtended != NULL;
 }
 
-void J9Ext::tryForceKeepJMethodIDs() {
```

Comment:
did this not work as expected?

Reply:
No :(
In one of the followup PRs in J9 they changed the approach to only mark classloaders of classes in frames handed out by ASGCT to retain jmethodids. And since the jmethodid crash is random, I got lucky for a few runs to be fooled into thinking it did :(